tests/test_sorting.py

```diff
@@ -32,6 +33,9 @@ def test_stooge_sort(self):
 
     def test_cocktail_shaker_sort(self):
         self.assertEqual([-44, 1, 2, 3, 7, 19], cocktail_shaker_sort([7, 3, 2, 19, -44, 1]))
+    
+    def tree_sort(self):
+	self.assertEqual([-44, 1, 2, 3, 7, 19], tree_sort([7, 3, 2, 19, -44, 1]))
```

Comment:
Are you comparing a tree with an array? Remember this is a library, it should be returned, not printed. 

- Also for this case, we may need documentation for the `insert()` function